sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,12 @@
+select  "name" as "genre name",
+        count("genres"."name") as "genres names"
+from    "genres"
+
+join    "filmGenre" using ("genreId")
+join    "castMembers" using ("filmId")
+join    "films" using ("filmId")
+join    "actors" using ("actorId")
+
+where   "actorId" = '178'
+
+group by "genres"."name";
```

Comment:
For this query, please restrict what information you use to that which you know ahead of time, without performing another query (manual or otherwise):

---

![image](https://user-images.githubusercontent.com/16143771/186548101-9526c03c-1239-4b79-b493-04c36611d961.png)